.github/workflows/build.yml

```diff
@@ -195,12 +196,13 @@ jobs:
            # release the app after building
            release: ${{ startsWith(github.ref, 'refs/tags/v') }}
 
-           # Skip auto-update files on branches other than main.
+           # Only publish auto-update files on v1.* stable.
            # Ideally we could use electron-builder release channels, but they don't work for github publishing.
-           # Auto-update is also disabled in code inside athens.main.core.
+           # Instead we control whether the auto-update is published (here) and whether the app will auto-update
+           # from releases or releases+prereleases (in athens.main.core).
            # NB: GH actions doesn't have ternary operator, but can workaround using AND/OR logic.
            # see https://github.com/actions/runner/issues/409#issuecomment-752775072
-           args: ${{ github.ref != 'refs/heads/main' && '-c.publish.publishAutoUpdate=false' || '' }} ${{ matrix.electron-arch-overrides }}
+           args: ${{ startsWith(github.ref, 'refs/tags/v1.') && '-c.publish.publishAutoUpdate=true' || '-c.publish.publishAutoUpdate=false' }} ${{ matrix.electron-arch-overrides }}
```

Comment:
This doesn't take into account suffixes, like `*-beta-rtc`, does it?

Reply:
Correct, it just determines if update data should be published. Here we are saying "publish update data if v1.*".